src/region/transformUtils.ts

```diff
@@ -0,0 +1,24 @@
+export type Point = {
+    x: number;
+    y: number;
+};
+
+export const invertYCoordinate = ({ x, y }: Point, height: number): Point => ({
+    x,
+    y: height > 0 ? height - y : y,
+});
+
+export const rotatePoint = ({ x, y }: Point, rotationInDegrees: number): Point => {
+    const { cos, sin } = Math;
+    const angle = (rotationInDegrees * Math.PI) / 180;
+
+    return {
+        x: x * cos(angle) - y * sin(angle),
```

Comment:
Seems like variables for these calculations would make sense: `const cosine = Math.cos(angle)`?